packages/cli/src/controllers/oauth/oAuth1Credential.controller.ts

```diff
@@ -103,31 +103,31 @@ export class OAuth1CredentialController extends AbstractOAuthController {
 	/** Verify and store app code. Generate access tokens and store for respective credential */
 	@Get('/callback', { usesTemplates: true })
 	async handleCallback(req: OAuthRequest.OAuth1Credential.Callback, res: Response) {
+		const userId = req.user?.id;
 		try {
-			const { oauth_verifier, oauth_token, cid: credentialId } = req.query;
+			const { oauth_verifier, oauth_token, state: encodedState } = req.query;
 
-			if (!oauth_verifier || !oauth_token) {
-				const errorResponse = new ServiceUnavailableError(
-					`Insufficient parameters for OAuth1 callback. Received following query parameters: ${JSON.stringify(
-						req.query,
-					)}`,
+			if (!oauth_verifier || !oauth_token || !encodedState) {
+				return this.renderCallbackError(
+					res,
+					'Insufficient parameters for OAuth1 callback.',
+					`Received following query parameters: ${JSON.stringify(req.query)}`,
 				);
-				this.logger.error('OAuth1 callback failed because of insufficient parameters received', {
-					userId: req.user?.id,
-					credentialId,
-				});
```

Comment:
This was going to sentry before. Now it's not. I assume it was not useful to have it in sentry?

Reply:
this would just create noise on sentry. without the user's complete setup reproduced locally, we would not be able to reproduce/fix this.